src/backend/search.go

```
@@ -42,7 +42,8 @@
 		})
 	}
 
-	tmpl, err := template.ParseFiles(templatePath + "search.html")
+	tmpl, err := template.ParseFiles(templatePath+"layout.html", templatePath+"search.html")
+	tmpl.ExecuteTemplate(w, "layout", searchResults)
```

golangci-lint (1.64.8)

Error return value of `tmpl.ExecuteTemplate` is not checked

(errcheck)